src/components/Card/src/Card/Card.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import MaterialCard from '@material-ui/core/Card';
+import { StylesProvider } from '@material-ui/core/styles';
+import { ArrowRightIcon } from '@gemeente-denhaag/icons';
+import BaseProps from '@gemeente-denhaag/baseprops';
+import './mui-override.css';
+import './card.css';
+import { Typography } from '@gemeente-denhaag/typography';
+import {
+  cardClasses,
+  cardArrowClasses,
+  cardTitleClasses,
+  cardSubtitleClasses,
+  cardCaseClasses
+} from './bem-mapping';
+import {
+  CardContent,
+  CardActions
+} from '../';
+import { Icon } from '@material-ui/core';
+
+export interface CardProps extends BaseProps {
+  /**
+   * Simple click handler
+   */
+  onClick?: () => void;
```

Comment:
This should contain a React event. Take a look at the changes @DanielvanVliet made to the IconButton props.